assets/stylesheets/units/_buttons.scss

```diff
@@ -0,0 +1,43 @@
+.button {
+  @include bold-font(20);
+
+  -webkit-appearance: none;
+  background-color: shade($green, 12.5%);
+  border: 0;
+  color: $white;
+  cursor: pointer;
+  display: inline-block;
+  letter-spacing: .5px;
+  margin: 0 ($baseline-grid-unit * 6) ($baseline-grid-unit * 3) 0;
+  padding: ($baseline-grid-unit * 3) ($baseline-grid-unit * 6);
+  text-align: center;
+  width: 100%;
+
+  @include media(tablet) {
+    width: auto;
+  }
+
+  &:link,
+  &:hover,
+  &:focus,
+  &:visited {
+    background-color: shade($green, 12.5%);
```

Comment:
Yes. In the first instance, I'll rework the colour here into a variable at the very least.